code/modules/surgery/organs/_organ.dm

```diff
@@ -251,12 +251,14 @@ INITIALIZE_IMMEDIATE(/obj/item/organ)
 	damage = clamp(damage + damage_amount, 0, maximum)
 	SEND_SIGNAL(src, COMSIG_ORGAN_ADJUST_DAMAGE, damage_amount, maximum, required_organ_flag)
 	. = (prev_damage - damage) // return net damage
-	var/message = check_damage_thresholds(owner)
+	var/message = check_damage_thresholds()
 	prev_damage = damage
 
+	var/was_failing = organ_flags & ORGAN_FAILING
 	if(damage >= maxHealth)
-		organ_flags |= ORGAN_FAILING
-	else
+		if(!was_failing)
+			organ_flags |= ORGAN_FAILING
```

Comment:
This `was_failing` check is redundant, `|=` won't do anything if it already has the failing flag